.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,55 @@
+## Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
Is there a way to automatically paste the commit message here?  I'd rather enforce good commit message habits and that should serve double duty as the PR description.

Reply:
I'm afraid it's not possible, I haven't found any options for placeholders in Github PR templates.
I'm afraid copy/paste is the only option if we go with templates.